tst/testinstall/perm.tst

```diff
@@ -487,6 +487,19 @@ gap> OnSets([(3,4),(1,2)],(1,2,70000));
 gap> OnSets([(1,2),(1,3)],(1,2,70000));
 [ (2,3), (2,70000) ]
 
+#
+# OnTuples and OnSets for a non-internal list
+#
+gap> r:=NewCategory("ListTestObject",IsSmallList);;
+gap> t:=NewType(ListsFamily, r and IsMutable and IsPositionalObjectRep);;
+gap> InstallMethod(ELM0_LIST,[r,IsPosInt],{l,i}->l![i]);
+gap> InstallMethod(Length,[r],l->3); # hard code length
+gap> l:=Objectify(t,[1,2,4]);;
+gap> OnTuples(l, (1,3));
+[ 3, 2, 4 ]
+gap> OnSets(l, (1,3));
+Error, Panic: cannot convert <list> (is a positional object) to a plain list
```

Comment:
Ah, this is caused by `IsSet` being called by `FuncOnSets`. But that is fixed in PR #3871; so let's first merge that PR, and then also PR #3877, after which I can rebase this PR and adjust the test output above, and also remove the references to `ELM0_LIST`